runtime/onert/core/include/ir/operation/FusedBatchNorm.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __ONERT_IR_OPERATION_FUSEDBATCHNORM_H__
+#define __ONERT_IR_OPERATION_FUSEDBATCHNORM_H__
+
+#include "ir/Operation.h"
+
+namespace onert
+{
+namespace ir
+{
+namespace operation
+{
+
+class FusedBatchNorm : public Operation
+{
+public:
+  enum Input
+  {
+    INPUT_X = 0,
+    INPUT_SCALE = 1,
+    INPUT_OFFSET = 2,
+    INPUT_EPSILON = 3,
+    INPUT_MEAN = 4,
+    INPUT_VARIANCE = 5
+  };
```

Comment:
As I know, our internal tflite model file's `CUSTOM(FusedBatchNorm)` has 5 inputs and 6 outputs. Could you tell us where do can we find this input/output size and role?

Reply:
I will revised input/output to 5/6 by referring to the official document of tensorflow.